app/models/user.rb

```diff
@@ -0,0 +1,7 @@
+class User < ApplicationRecord
+  validates :name, :email, :password, :phone, :user_name, :role, presence: true
+  validates :phone, length: {minimum: 11, maximum: 15}, numericality: { only_integer: true }
```

Comment:
introduce phony rails gem

Reply:
Apu used phony rails for mobile validation.